nodeCf.js

```diff
@@ -1,176 +1,207 @@
-var Promise = require('bluebird');
-var nunjucks = require("nunjucks");
-var _ = require('lodash');
-var Ajv = require('ajv');
-var fs = require('fs');
+const Promise = require('bluebird');
```

Comment:
Doesn't look like these are supported out of the box with node v7.9.0 yet
https://medium.com/the-node-js-collection/an-update-on-es6-modules-in-node-js-42c958b890c